pxr/usdImaging/usdviewq/appController.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, mainWindow, parent, name):
                 default=[True, True, True, True, False], validator=lambda value: 
                 len(value) == 5)
         propertyViewColumnVisibility = self.stateProperty("propertyViewColumnVisibility",
-                default=[True, True, True], validator=lambda value: len(value) == 3)
```

Comment:
There is, but it looked more invasive. The attributes in the property view aren't backed by a model, and I didn't see one already in the code, the attributes are added by `addChild` calls.

Reply:
If I'm reading things correctly, you're normalizing when you match, so is it even necessary to store?

I'd recommend we change `lower` to `casefold` as well since `casefold` is designed for matching.

Reply:
The normalization during match only happens for prims. The attributes are searched by calling `QTreeWidget.findItems`, which doesn't accept a predicate. It does accept a regular expression though, I can try experimenting to see if there's a regex that can match both unicode and non unicode.

Reply:
If this just needed for properties then, I think it's reasonable to give each item a normalized name for matching data role.

Reply:
Ok, I'll need to change the property view from a QTreeWidget to a QTreeView in order to use a model. The widget is a higher level control that doesn't expose the model.

Reply:
I changed this to add the normalized name as data to the existing Name column and use the models' match on that.